blockscout-ens/graph-node/subgraphs/d3-connect-subgraph/networks.json

```diff
@@ -8,8 +8,12 @@
   },
   "shibarium-testnet": {
     "Registry": {
-      "address": "0x91c2d22ca1028B2E55e3097096494Eb34b7fc81c",
-      "startBlock": 4555130
+      "address": "0x1443bC2bBAB07437BCF9C577b647523A736bB33E",
+      "startBlock": 1927346
+    },
+    "Resolver": {
```

Comment:
To deploy this subgraph on shibarium-mainnet, we need to specify Resolver for mainnet also

Reply:
Once the new mainnet resolvers are deployed I'll update them.